### DISCUSSION TITLE
BUG: SumatraPDF cannot open a *.fb2.zip file, if it contains other files in the archive

### DISCUSSION THREAD

**Discussion by andrey-zhmakin** [sumatrapdfreader/sumatrapdf#3322] [Q&A]
Consider this file: https://royallib.com/get/fb2/chertok_boris/raketi_i_lyudi_goryachie_dni_holodnoy_voyni.zip
When you try to download it, it will rename itself "Черток Борис. Книга 1. Ракеты и люди - royallib.com.fb2.zip" somehow. But the name does not matter. Make sure that the file name ends with ".fb2.zip" for the actual file on your disk.

SumatraPDF both 3.4.6 (64-bit) and pre-release build version 15330, built on 2023-03-02 , commit [0f865edef5995917c9131dc80c7dbbe5a1a602db](https://github.com/sumatrapdfreader/sumatrapdf/commit/0f865edef5995917c9131dc80c7dbbe5a1a602db)

fails to open this file.

Older versions of SumatraPDF did work. In fact I reported this bug long time ago and it had been subsequently fixed.

The problem is that the file in question contains more than one file inside. If you delete files other than .fb from the archive, SumatraPDF will open it just fine.

Calibre opens this file without any problems.

**Comment by andrey-zhmakin** [comment]
Maybe I should delete this discussion and create an issue instead?

**Reply by GitHubRulesOK** [reply]
I think the issue is possibly covered by more than one already. 
as a zip file it may cause one issue, of "blocker" file names
 and as a fb2.zip a separate issue about detection as a fb2z from any other docX file type, too many files start with zip ID markings (PK)
